mobilecoind-json/src/data_types.rs

```diff
@@ -837,11 +837,33 @@ impl From<&RingMLSAG> for JsonRingMLSAG {
     }
 }
 
+// Representa TokenId (u64) using string, when serializing to Json
+// This does not rely on the serde-json arbitrary precision feature, which
+// (we fear) might break other things
+#[derive(Deserialize, Serialize, Default, Debug, Clone)]
+#[serde(transparent)]
+pub struct JsonTokenId(#[serde(with = "serde_with::rust::display_fromstr")] pub u64);
```

Comment:
@eranrund @remoun @jcape lmk if you guys are okay with this fix

Reply:
I think this is basically the same as "just using strings" in the json, it's just less typing

We could change this to `JsonU64` I guess and use it for other u64s